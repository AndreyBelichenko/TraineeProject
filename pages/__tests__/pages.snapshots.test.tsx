import * as React from 'react';
import AccountComponent from '../account';
import Auth from '../auth';
import ListComponent from '../list';
import { configure, shallow } from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
const adapter = Adapter as any;
configure({ adapter: new adapter() });

describe('testing pages', () => {
  let tree: any;
  it('AccountComponent snapshot', () => {
    tree = shallow(<AccountComponent />);
    expect(tree).toMatchSnapshot();
  });

  it('Auth snapshot', () => {
    tree = shallow(<Auth />);
    expect(tree).toMatchSnapshot();
  });

  it('ListComponent snapshot', () => {
    tree = shallow(<ListComponent />);
    expect(tree).toMatchSnapshot();
  });
});
