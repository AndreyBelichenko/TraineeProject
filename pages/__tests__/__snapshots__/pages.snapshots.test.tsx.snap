// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing pages AccountComponent snapshot 1`] = `ShallowWrapper {}`;

exports[`testing pages Auth snapshot 1`] = `ShallowWrapper {}`;

exports[`testing pages ListComponent snapshot 1`] = `ShallowWrapper {}`;
